client/src/main/java/io/obswebsocket/community/client/message/request/transitions/SetTbarPositionRequest.java

```diff
@@ -27,7 +27,6 @@ private SetTbarPositionRequest(Double position, Boolean release) {
     @NonNull
     private final Double position;
     @NonNull
-    @Builder.Default
-    private final Boolean release = true;
+    private final Boolean release;
```

Comment:
Can you explain why that change?

Reply:
because it is unnecessary, in the constructer we always provide the value of it, its ether `null` or a boolean value.